packages/apps-config/src/api/spec/index.ts

```diff
@@ -267,6 +268,8 @@ const spec: Record<string, OverrideBundleDefinition> = {
   subspace,
   'substrate-contracts-node': substrateContractsNode,
   subzero: zero,
+  supersig,
+  'node-template': supersig,
```

Comment:
This doesn’t work. Node-template is a standard with the base Substrate types. Please remove.